src/server/index.js

```diff
@@ -23,7 +21,7 @@ function startServer(state) {
 
   // Generic server errors (e.g. not caught by components)
   server.use((err, req, res, next) => {
-    winston.error(req.method + ' ' + req.url, {error: err})
+    console.error(req.method + ' ' + req.url, {error: err})
```

Comment:
While dropping the logger altogether?

Seems like it add a purpose: https://github.com/openfisca/legislation-explorer/pull/93

Do you think it's just to complex for what it brings?

Reply:
It is only used in this single occurrence.
#93 was about moving away from email-based error notifications that were never set up properly and that no one ever read. I was asked by the implementer if I had suggestions on how to log and recommended winston.
Adding a logger-dedicated module only makes sense if we actually use it. Currently, there is no monitoring in place, and the only usage of the error-logging mechanism was this line of code. It will be trivial to revert when / if someone puts the effort to set up logging properly. On the other hand, having it right now means adding a (useless) dependency and a lot of config complexity in order to pass a config object.